tests/hypertext/HTMLLintBearTest.py

```diff
@@ -12,18 +12,34 @@
 HTMLLintBearTest = verify_local_bear(HTMLLintBear,
                                      valid_files=(),
                                      invalid_files=(test_file,),
+                                     settings={'use_spaces': 'False'},
                                      tempfile_kwargs={'suffix': '.html'})
 
 HTMLLintBearIgnoreTest = verify_local_bear(
     HTMLLintBear,
     valid_files=(test_file,),
     invalid_files=(),
-    settings={'htmllint_ignore': 'optional_tag'},
+    settings={'use_spaces': 'False', 'htmllint_ignore': 'optional_tag'},
     tempfile_kwargs={'suffix': '.html'})
 
 HTMLLintBearIgnoreQuotationTest = verify_local_bear(
     HTMLLintBear,
     valid_files=(),
     invalid_files=(test_file,),
-    settings={'htmllint_ignore': 'quotation'},
+    settings={'use_spaces': 'False', 'htmllint_ignore': 'quotation'},
+    tempfile_kwargs={'suffix': '.html'})
+
+test_ignore_tabs = """
```

Comment:
I think it it would be a better idea if you keep your files together (at the top with test_file) :)